server/server/src/logging.rs

```diff
@@ -18,8 +19,32 @@ pub fn logging_init(settings: Option<LoggingSettings>) {
         LogMode::Console => LogConfig::new().console(),
         LogMode::All => file_logger(&settings).console(),
     };
-    fast_log::init(config.level(LevelFilter::from(settings.level.clone())))
-        .expect("Unable to initialise logger");
+    fast_log::init(
+        config
+            .level(LevelFilter::from(settings.level.clone()))
+            .custom(NativeLogger {}),
+    )
+    .expect("Unable to initialise logger");
+    log::warn!("omSupply Logging initialised aaabbbccc");
+}
+
+struct NativeLogger {}
+impl LogAppender for NativeLogger {
+    fn do_logs(&self, records: &[FastLogRecord]) {
```

Comment:
You could have a logging_init signature like:

```rust
pub fn logging_init<Appender: LogAppender + 'static>(settings: Option<LoggingSettings>, arg: Option<Appender>)
```

and then pass in the Android logcat logger in the android crate, i.e. no android dependencies in the server crate. (Untested so there might be some pitfalls)